store/tikv/lock_resolver.go

```diff
@@ -33,6 +33,9 @@ import (
 // ResolvedCacheSize is max number of cached txn status.
 const ResolvedCacheSize = 2048
 
+// Transaction which involve keys exceed this threshold can treat as `Big Transaction`.
+const BigTxnThreshold = 256
```

Comment:
I think it is too large. A cop request may need 200+ round trips to resolve lock.

Reply:
Yes, i change the txn_size as keys_in_region, it is more accurate.